Documentation.md

```diff
@@ -85,7 +83,6 @@ If we all chip in with documentation we can help set everyone up for success.
 
 ## Links for HXers
 
-*   [dockyard metrics](https://services-hub.dock-yard.io/)
-*   [team roadmaps](https://sites.google.com/holidayextras.com/journey/home)
-*   [learning and development hub](https://sites.google.com/holidayextras.com/tech-team-ld-hub)
-*   [agile processes hub ](https://sites.google.com/holidayextras.com/hx-agile-coaches/home)
+*   [Dockyard metrics](https://services-hub.dock-yard.io/)
```

Comment:
Should we remove this link? This culture is public but the link only works internally so this isn't relevant to all audiences